lib/salus/scanners/base.rb

```diff
@@ -53,6 +53,10 @@ def self.supported_languages
       []
     end
 
+    def self.scanner_type
```

Comment:
```ruby
def self.scanner_type
  raise NoMethodError, 'implement in subclass'
end
```

Usually, in ruby, we force the developers to implement the method in subcalss by raising an exception. We have similar code in other [places](https://github.com/coinbase/salus/blob/9c93eb2e2b9d87b4a60de7bc1512c4f0556772ce/lib/salus/scanners/node_audit.rb#L143).

Reply:
Done